.travis.yml

```diff
@@ -18,6 +18,7 @@ script:
 - python kalite/manage.py syncdb --migrate --noinput --traceback
 - python kalite/manage.py test --traceback -v 2
 - grunt jshint
+- ./node_modules/karma/bin/karma start --single-run --browsers PhantomJS
```

Comment:
I think doing it this way means you need to install `karma` locally, not with `-g`. See how `grunt` is invoked above.